docs/content/docs/features/primary-key-table.md

```diff
@@ -158,12 +154,20 @@ Full compaction changelog producer can produce complete changelog for any type o
 
 {{< /hint >}}
 
-## Changelog Tables without Primary Keys
+## Sequence Field
 
-Changelog tables can also be used without primary keys. Users can only insert or delete a whole record from the table. No update is supported.
+By default, the primary key table determines the merge order according to the input order. However, in distributed computing,
+there will be some cases that lead to data disorder. At this time, you can use a time field as `sequence.field`, for example:
 
-## Append-only Tables
-
-By specifying `'write-mode' = 'append-only'` when creating the table, user creates an append-only table.
+```sql
+CREATE TABLE MyTable (
+    pk BIGINT PRIMARY KEY NOT ENFORCED,
+    v1 DOUBLE,
+    v2 BIGINT,
+    dt TIMESTAMP
+) WITH (
+    'sequence.field' = 'dt'
+);
+```
 
-You can only insert a whole record into the table. No delete or update is supported and you cannot define primary keys. This type of table is suitable for use cases that do not require updates (such as log data synchronization).
+Regardless of the sequence of data input, the final correct result will always be obtained.
```

Comment:
Which is the final correct result? Do you mean that the output data of the table is correct no matter what the sequence of the data input is.

Reply:
I change this to: `Regardless of the order of data input, the final correct result will always be obtained`.